docs/vector-indexes.md

```diff
@@ -59,10 +59,11 @@ class MyPage(VectorIndexedMixin, Page):
     vector_index_class = MyEmbeddableFieldsVectorIndex
 ```
 
+NOTE: Where `vector_index_class` is set on a subclass of `VectorIndexedMixin` this way, the index will be registered with `wagtail-vector-index` automatically. In more complex cases, you may need to register your index class manually (read on for an example).
```

Comment:
@tomusher I've added this note instead of moving the manual registration example here.